defaults.go

```diff
@@ -29,25 +28,6 @@ func RegisterPodDefault(name string, callback NewPodDefaultFunc) error {
 
 func init() {
 	RegisterPodDefault("minio-mounts", func(profile *kubeflowv1.Profile) *kubeflowv1alpha1.PodDefault {
```

Comment:
Should this still be here? References minio? This file itself might be useless / not needed for now it looks like, but may be helpful for sure in the future if we need some of these kinds of configs again.

In which case, 
do we even want this `configurations` tab anymore if its empty?